app/client/src/features/accounts/UserSideNav.tsx

```diff
@@ -88,11 +92,22 @@ export function UserSideNavLoader() {
 }
 
 export function UserSideNav({ queryRef, onClick }: UserSideNavProps) {
+    // const { isModerator } = useEvent();
+    const data = usePreloadedQuery<UserSideNavQuery>(USER_SIDE_NAV_QUERY, queryRef);
+
     const classes = useStyles();
     const router = useRouter();
     usePreloadedQuery(USER_SIDE_NAV_QUERY, queryRef);
     const [selected, setSelected] = React.useState<Keys | undefined>(findTab(router.pathname));
 
+    const { displaySnack } = useSnack();
+    React.useEffect(() => {
+        if (!data.isOrganizer && router.pathname === '/organizations/me') {
+            router.push('/dashboard');
+            displaySnack('Organizers-only path!');
```

Comment:
I would say "not allowed." We would want to leak unnecessary permission policies.

Reply:
With the new role guard feature, we don't need this code at all so I commented it all out for now, but I still did change the string to "not allowed"!